src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.ImportWorkloads.targets

```diff
@@ -35,4 +35,27 @@ Copyright (c) .NET Foundation. All rights reserved.
                           NETCoreSdkVersion="$(NETCoreSdkVersion)"
                           GenerateErrorsForMissingWorkloads="true"/>
   </Target>
+
+  <!--  This target is not part of the build. Only used by dotnet workload restore command. Global property "SkipResolvePackageAssets"
+        need to be set to "true" to avoid requiring restore.-->
+  <Target Name="_FindAllReferenceWorkload" DependsOnTargets="GetSuggestedWorkloads;PrepareProjectReferences" Returns="@(_ResolvedSuggestedWorkloads)">
+    <MSBuild
+      Projects="@(_MSBuildProjectReferenceExistent)"
+      Targets="GetSuggestedWorkloads"
+      BuildInParallel="$(BuildInParallel)"
+      Properties="GenerateErrorsForMissingWorkloads=false; SkipResolvePackageAssets=true"
```

Comment:
I don't think you need to set either of these properties here.  `SkipResolvePackageAssets` should already be set as a global property, which means it should flow through to project references.  And `GenerateErrorsForMissingWorkloads` is a parameter to the `ShowMissingWorkloads` task, but setting it as a property doesn't do anything.